### PR TITLE
longvol generic reader fix

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
The generic reader was set to deal with longvol on 32 bits images. This PR fix it with the update of test.

*your description here*

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [n.a.] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/io/readers/GenericReader.ih [hunk 1]
```diff
@@ -136,6 +136,26 @@ import( const std::string & filename,
...
+template <typename TContainer>
+inline
+TContainer
+DGtal::GenericReader<TContainer, 3, DGtal::uint64_t>::
+import( const std::string & filename) throw( DGtal::IOException )
+{
+  DGtal::IOException dgtalio;
+  const std::string extension = filename.substr( filename.find_last_of(".") + 1 );
+
+  if ( extension == "longvol" )
```
Comment (dcoeurjo) [inline src/DGtal/io/readers/GenericReader.ih]: I use sometime "lvol" for longvol extension.. I don't know if it is "standard"..

File: tests/io/readers/testGenericReader.cpp [hunk 2]
```diff
@@ -59,8 +59,10 @@ bool testGenericReader()
   typedef DGtal::ImageContainerBySTLVector<DGtal::Z2i::Domain, unsigned char> Image2D;
 
   typedef DGtal::ImageContainerBySTLVector<DGtal::Z3i::Domain, unsigned int> Image3D32bits;
```
Comment (dcoeurjo) [inline tests/io/readers/testGenericReader.cpp]: unsigned int -> uint32_t to be consistent

File: src/DGtal/io/readers/GenericReader.ih (hunk 1 above)
Comment (kerautret) [inline src/DGtal/io/readers/GenericReader.ih]: ok I add this possibility

**Comment by kerautret** [comment]
@dcoeurjo done thanks

File: src/DGtal/io/readers/GenericReader.ih [hunk 3]
```diff
@@ -118,7 +118,7 @@ import( const std::string & filename,
   DGtal::IOException dgtalio;
   const std::string extension = filename.substr( filename.find_last_of(".") + 1 );
 
-  if ( extension == "longvol" )
+  if ( extension == "longvol" || extension == "lvol" )
```
Comment (dcoeurjo) [inline src/DGtal/io/readers/GenericReader.ih]: thanks;)

File: ChangeLog.md [hunk 4]
```diff
@@ -29,6 +29,10 @@
    the issue. (Bertrand Kerautret
    [#1130](https://github.com/DGtal-team/DGtal/pull/1130))
 
+ - readers: include longvol reader in GenericReader for 64 bits images. Update
```
Comment (dcoeurjo) [inline ChangeLog.md]: 64bit images ? (instead of bits)

Comment (kerautret) [inline ChangeLog.md]: yes done

**Comment by dcoeurjo** [comment]
oops, the branch cannot be merged anymore in github. Can you please merge the master back?

**Comment by kerautret** [comment]
yes i look it

**Comment by kerautret** [comment]
@dcoeurjo done

**Comment by dcoeurjo** [comment]
ok thanks.

**Pull request merged**